tools/scoring/score/parse_config.py

```diff
@@ -82,6 +84,7 @@ def append_types(
         The deserialized files dictionary with types
         appended to each entity in the respective files.
     """
+    print('Scoring — appending entity types')
```

Comment:
I dont think this is needed? it is an implementation detail?

Reply:
Fair point.

Reply:
Well, it contextualizes that follows, which _is_ relevant.